.github/workflows/docker-test.yml

```diff
@@ -8,13 +8,24 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - name: checkout code
-        uses: actions/checkout@v2
-      - name: install buildx
+      - name: Checkout
```

Comment:
is this file even needed, I don't see any usages ?

Reply:
Yup, this is being used for the Docker CI test